pwa/src/apiService/resources/organizations.ts

```diff
@@ -0,0 +1,59 @@
+import { Send } from "../apiService";
+import { AxiosInstance } from "axios";
+import { IFilters } from "../../context/filters";
+
+export default class Organization {
+  private _instance: AxiosInstance;
+
+  constructor(_instance: AxiosInstance) {
+    this._instance = _instance;
+  }
+
+  public getOne = async (id: string): Promise<any> => {
+    const { data } = await Send(this._instance, "GET", `/organizations/${id}`);
+
+    return data;
+  };
+
+  public getAll = async (filters: IFilters): Promise<any> => {
+    const { data } = await Send(
+      this._instance,
+      "GET",
+      `/organizations?page=${filters.currentPage}&limit=10&extend[]=all${filtersToQueryParams(filters)}`,
+    );
+
+    return data;
+  };
+
+  public getCount = async (filters: IFilters): Promise<any> => {
+    const { data } = await Send(this._instance, "GET", `/organizations?limit=1${filtersToQueryParams(filters)}`);
+
+    return data.total;
+  };
+}
+
+const filtersToQueryParams = (filters: any): string => {
+  delete filters.resultDisplayLayout;
+
+  let params: string = "";
+
+  for (const [key, value] of Object.entries(filters)) {
+    if (!value) continue;
+
+    if (typeof value === "string") {
+      params += `&${key}=${value}`;
+    }
+
+    if (Array.isArray(value)) {
+      let arrayParams = "";
+
+      value.forEach((value) => {
+        arrayParams += `&${key}[]=${value}`;
+      });
+
+      params += arrayParams;
+    }
+  }
+
+  return params;
+};
```

Comment:
This (seems to be) is a duplicate piece of logic which we're also using within the component resource. I see that the only difference is the filters we're deleting before setting the params.

I suggest creating a generic service to do this, to which you send the already _filtered_ filters object (so we do the deletions **before** passing them to the `filtersToQueryParams` function).

This way, the function can be generic, and _what_ we want to be added to the query params can be filtered by the implementing context.